docs/reference/feature-servers/python-feature-server.md

```diff
@@ -219,7 +219,7 @@ The above command will generate two files
 To start the feature server in SSL mode, you need to provide the private and public keys using the `--ssl-key-path` and `--ssl-cert-path` arguments with the `feast serve` command.
 
 ```shell
-feast serve --ssl-key-path key.pem --ssl-cert-path cert.pem
+feast serve --ssl-key-path /path/to/key.pem --ssl-cert-path /path/to/cert.pem
```

Comment:
Nit: this name is a little misleading

Reply:
The standard k8s secrets for TLS have 2 keys `tls.crt` and `tls.key`
These can be generated using the CLI with:
```
kubectl create secret tls tls-secret --cert=path/to/tls.crt --key=path/to/tls.key
```
Can we use the same options? [Refs](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret_tls)

Reply:
@franciscojavierarceo - what do you think of those options? `ssl_filename` can be used for private key but needs to come up with other name for public key. `pem_filename` private key extension can be different from pem as well. If you are ok with Daneil's proposal then i can update the code for this PR and also online server code to keep it consistent across feast.

Reply:
+1 to @dmartinol's suggestion

Reply:
Incorporated this change. Please take a look at it again.